packages/core/src/button/Button.css

```diff
@@ -115,7 +116,6 @@
   -webkit-tap-highlight-color: transparent;
   /* Styles applied to align children*/
   font-weight: var(--saltButton-fontWeight, var(--button-fontWeight));
-  white-space: pre;
```

Comment:
Removing this means users don't need to remove the space between the icon and text.